.travis.yml

```diff
@@ -17,8 +17,17 @@ before_install:
 install:
   - ./setup.py install
 
+before_script:
+  - mkdir exec
+  - cd exec
+  - git clone https://github.com/fossology/fossology
+  - git clone https://github.com/nexB/scancode-toolkit
```

Comment:
I would move this line after the line `make -f Makefile.sa`.
Furthermore, instead of cloning the repo, it is probably better to install a specific version (instructions [here](https://github.com/nexB/scancode-toolkit#installation))

something like the lines below should be enough
```
wget https://github.com/nexB/scancode-toolkit/releases/download/v3.0.0/scancode-toolkit-3.0.0.zip
unzip scancode-toolkit-3.0.0.zip
```